validator/beaconnode/src/main/java/tech/pegasys/teku/validator/beaconnode/TimeBasedEventAdapter.java

```diff
@@ -87,7 +88,7 @@ private boolean isTooLate(final UInt64 scheduledTime, final UInt64 actualTime) {
   public SafeFuture<Void> start() {
     // Don't wait for the genesis time to be available before considering startup complete
     // The beacon node may not be available or genesis may not yet be known.
-    genesisTimeProvider.getGenesisTime().thenAccept(this::start).reportExceptions();
+    genesisDataProvider.getGenesisData().thenAccept(this::start).reportExceptions();
```

Comment:
Could this continue using `genesisDataProvider.getGenesisTime()`?

Reply:
yep it got caught in a rename